compiler/rustc_hir/src/hir.rs

```diff
@@ -2057,6 +2057,26 @@ pub enum PrimTy {
 }
 
 impl PrimTy {
+    pub const ALL: [Self; 17] = [
```

Comment:
Please leave a comment on this mentioning that any changes should also be reflected in `from_name`. Also leave the opposite comment on `from_name`.